lib/src/analyzer/extractor.dart

```diff
@@ -70,3 +81,45 @@ Future<List<Declaration>> extractDeclarations(String path) async {
 
   return visitedDeclarations.values.toList();
 }
+
+List<(Declaration, List<int>)> extractUntestedDeclarations(
+  Map<String, List<Declaration>> declarations,
+  CoverageData coverageResults,
+) {
+  final untestedDeclarations = <(Declaration, List<int>)>[];
+
+  for (final (filePath, uncoveredLines) in coverageResults) {
+    final fileDeclarations = declarations[filePath] ?? [];
+    for (final declaration in fileDeclarations) {
+      final lines = <int>[];
+      for (final line in uncoveredLines) {
+        if (line >= declaration.startLine && line <= declaration.endLine) {
+          lines.add(line - declaration.startLine);
+        }
+      }
+      if (lines.isNotEmpty) {
+        untestedDeclarations.add((declaration, lines));
+      }
+    }
+  }
+
+  return untestedDeclarations;
+}
+
+/// Converts an absolute file path into a package import path
+String _toPackageImportPath({
```

Comment:
Low priority: Use package_config instead of doing this manually.

https://github.com/AmrAhmed119/dart-testgen/issues/12

Reply:
For now you can just add a TODO